### ISSUE TITLE
Disable auto-selection on first row?

### ISSUE THREAD

**Issue by kgrzesiu** [akveo/ng2-smart-table#349]
Hi!
Is there a way to disable auto-selection and remove the possibility to select rows as well?

Didn't find it in the docu...

Cheers,
Greg

**Comment by lopenchi** [comment]
@kgrzesiu  do you find a solution?

**Comment by kgrzesiu** [comment]
@lopenchi No, still did not. A simple property for disable row selection would solve this, but I think there is none.

**Comment by amfsyn** [comment]
The property is st-select-row. add "st-select-row="fn(row)". If the row should be selectable have that function return the row.
